<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderManager</class>
 <widget class="QWidget" name="OrderManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Order Section</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="OrdertabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>&amp;Order</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="OrderClientlabel">
              <property name="text">
               <string>Client</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QComboBox" name="clientComboBox">
                <item>
                 <property name="text">
                  <string>ID</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Name</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Address</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Type</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="clientLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTreeWidget" name="clientTreeWidget">
              <column>
               <property name="text">
                <string>ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Address</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Tel</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Type</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="OrderClientlabel_2">
              <property name="text">
               <string>Item</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QComboBox" name="itemComboBox">
                <item>
                 <property name="text">
                  <string>ID</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Name</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Categori</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Color</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="itemLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTreeWidget" name="itemTreeWidget">
              <column>
               <property name="text">
                <string>ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Categori</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Color</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Stock</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Price</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="OrderClientlabel_3">
              <property name="text">
               <string>Order</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>&amp;Client ID</string>
                </property>
                <property name="buddy">
                 <cstring>orderClientIdlineEdit</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="orderClientIdlineEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>&amp;Item ID</string>
                </property>
                <property name="buddy">
                 <cstring>orderItemIdlineEdit</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="orderItemIdlineEdit"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>&amp;Price</string>
                </property>
                <property name="buddy">
                 <cstring>orderPricelineEdit</cstring>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="orderPricelineEdit"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>&amp;Quantity</string>
                </property>
                <property name="buddy">
                 <cstring>orderQuantitylineEdit</cstring>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="orderQuantitylineEdit"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>&amp;Amount</string>
                </property>
                <property name="buddy">
                 <cstring>orderAmountlineEdit</cstring>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="orderAmountlineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="clearPushButton">
                <property name="text">
                 <string>Clear</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="orderPushButton">
                  <property name="text">
                   <string>Order</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="modifyPushButton">
                  <property name="text">
                   <string>Modify</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="RemovePushButton">
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>&amp;Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="SearchLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Bahnschrift SemiBold</family>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Order Infomation Search</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>186</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QComboBox" name="SearchComboBox">
              <item>
               <property name="text">
                <string>Order Num</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Clietn ID</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Item ID</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Categori</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Color</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="searchLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="SearchPushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeWidget" name="searchTreeWidget">
            <column>
             <property name="text">
              <string>Order Num</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Client Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Item Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Quantity</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QListWidget" name="SearchTimeListWidget">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Order List</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QTreeWidget" name="orderTreeWidget">
           <column>
            <property name="text">
             <string>Order Num</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Client Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Item Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Quantity</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Amount</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="logTimeLineEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
